core/src/com/unciv/logic/map/MapParameters.kt

```diff
@@ -52,7 +52,7 @@ class MapParameters : IsPartOfGameInfoSerialization {
     var type = MapType.pangaea
     // DO NOT CHANGE DEFAULTS since that changes all existing games to new default!
```

Comment:
I remember really bad things happened with the default shape, so let's test what happens if you take a previously-Medium map and change this

Reply:
You mean create a radius 20 map ad see if it does weird, or from an existing save?

Reply:
From an existing save

Reply:
I don't notice anything being changed to by saves, do I need to look for something specifically?

Reply:
Most of them are Large map though, not Medium.